.betterer.results

```diff
@@ -2362,6 +2363,10 @@ exports[`better eslint`] = {
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "10"],
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "11"]
     ],
+    "public/app/features/alerting/unified/components/rule-editor/dag.ts:5381": [
```

Comment:
This one is for the custom type guard in `isDagError`

Reply:
I would argue that if they're going to stay like that/we don't plan to fix, we could `eslint-ignore` them instead?

Reply:
Oh I tried `ts-ignore` and that didn't work – didn't know this was being checked by `eslint` instead 😅

Reply:
Fixed by using error subclassing :)